websites/B/Battlefield Portal/dist/metadata.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
```

Comment:
Please tell me where you saw a presence/tutorial using v1.3 and, while you're at it, change this to 1.5
```suggestion
    "$schema": "https://schemas.premid.app/metadata/1.5",
```

Reply:
> Please tell me where you saw a presence/tutorial using v1.3 and, while you're at it, change this to 1.5

https://eggsy.xyz/projects/premid/mdcreator

Reply:
Thank you, will get it fixed